control/quadraframe_model.hh

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "control/jet_vane_model.hh"
+#include "control/wrench.hh"
+#include "control/vanes.hh"
+
+
+namespace jet {
+namespace control {
+
+struct QuadraframeConfiguration {
+  const jcc::Vec3 quadraframe_from_vane_unit_0_translation = jcc::Vec3(0.0, 46.7e-3, -9.5e-3);
```

Comment:
Put this location in a config file or grab from CAD?

Reply:
Will do CAD grab in a follow-up diff since it's a bit nontrivial